packages/terra-clinical-result/src/common/utils.js

```diff
@@ -35,17 +63,10 @@ const checkTypeNumeric = (resultData) => {
  */
 const ConditionalWrapper = ({ condition, wrapper, children }) => (condition ? wrapper(children) : children);
 
-const ClinicalResultUtils = {
-  isEmpty,
-  checkIsStatusInError,
-  checkTypeNumeric,
-  ConditionalWrapper,
-};
-
-export default ClinicalResultUtils;
 export {
   isEmpty,
   checkIsStatusInError,
   checkTypeNumeric,
+  createResult,
```

Comment:
jest test on this guy?

Reply:
Made https://github.com/cerner/terra-clinical/pull/648/commits/bcd9623974696cc69c750cf77514f41a424ac509#diff-8b312db510259892da0886d2f909d49eR89